scss/detail.scss

```diff
@@ -0,0 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+.titulo-libro{
+  color: black;
+  font-family: 'Roboto', sans-serif;
```

Comment:
This font-family rule is repetead in several classes, and I think it's bound to be repeated again.
How about making a class with low specificity with this rule so you don't have to repeat it in every class?

Reply:
i could make a "text" class or something similar and add the font-family there. But how do i use it then? can i create an element with 2 classes???  for example: <h1 class="book-title" "text">  what would be the right way to use that new low specificity class?

Reply:
If you make a text class you thanks to sass you can [use @extend](https://sass-lang.com/documentation/at-rules/extend) it to other classes that would use the same rules as `.text`

Yes, you can assign more than 2 classes to an element for example
```html
<p class="greeting alt-color">Hi</p>
```
Regarding low specifity this is a list of selectors in ascending order (lower specifity / null to max) [check this out!](https://specifishity.com/)